modules/hcp-ts-cidr-overlap.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-troubleshooting.adoc
+
+:_mod-docs-content-type: CONCEPT
```

Comment:
The presence of the .Procedure title and a list item with an instruction makes this more of a procedure than a concept module. I see two ways to resolve this discrepancy:

1. Change this content type to PROCEDURE and ideally find some way to add a gerund in the title, like "Resolving an OVN subnets overlap with CIDR"
2. Keep this as a concept module but remove the .Procedure and rewrite the imperative list item as some kind of plain paragraph without explicit instruction

Reply:
Oh my goodness -- I totally overlooked the content type on these modules! I will change the types to PROCEDURE and fix the headings to use gerunds. 🤦‍♀️